README.md

```diff
@@ -5,18 +5,31 @@
 
 # Egree
 
-Ruby client for the [Egree API](https://app.egree.com/apiv1).
+## You're on branch api/v2, if you want to use apiv1, checkout [master](https://github.com/kollegorna/egree-ruby/tree/master).
 
-Currently the only supported api calls is `createcasecommand` and `getviewcaseurlquery`.
+Ruby client for the [Egree API V2](https://app.egree.com/apiv2).
+
+Currently the only supported api calls are `createcase`, `sendcase` and `getcase`.
+
+### Note
+If you miss `getviewcaseurlquery`, this is how you can get the view case url.
+
+```
+signing_case = egree.get_case case_id
+signing_case.response["Parties"].first["PartyUrl"] # Each party gets their own url
+
+```
 
 ## Usage
 
 ### Creating a case
 
 ```ruby
-egree = Egree.client username, password
+egree = Egree.client egree_api_key, egree_api_secret
 
-signature_case = Egree::Case.new "Agreement", ["touch"]
+case_id = SecureRandom.uuid
```

Comment:
I guess you found the `Egree::ReferenceId` to be a bit over engineered =)
If we are promoting the use of `SecureRandom.uuid` directly instead I think we should drop `Egree::ReferenceId` completely.

It felt a bit silly when I added it and it was mainly for consistency with how the Serializers / ApiMappers work for other properties.